### PR TITLE
Disabled custom engine initialization until ready

### PR DESCRIPTION
**Pull request by nukeulater**
Added a few more offsets for dedicated server.

### CODE REVIEW

File: xlive/XLive/UserManagement/CUser.cpp [hunk 1]
```diff
@@ -427,8 +427,6 @@ void CUserManagement::ConfigureUser(XNADDR* pxna, ULONGLONG xuid, char* username
 	(*(DWORD*)&this->SecurityPacket[0]) = annoyance_factor;
 	(*(DWORD*)&this->SecurityPacket[4]) = pxna->inaOnline.s_addr;
 
-	SecurityPacket = this->SecurityPacket;
```
Comment (shayaantx) [inline xlive/XLive/UserManagement/CUser.cpp:430]: why is this being removed

File: xlive/H2MOD/Modules/Tweaks/Tweaks.cpp [hunk 2]
```diff
@@ -906,8 +906,8 @@ void InitH2Tweaks() {
 	addDebugText("Begin Startup Tweaks.");
 
 	MapChecksumSync::Init();
-	custom_game_engines::init();
-	custom_game_engines::register_engine(c_game_engine_types::unknown5, &g_test_engine, king_of_the_hill);
+	//custom_game_engines::init();
```
Comment (shayaantx) [inline xlive/H2MOD/Modules/Tweaks/Tweaks.cpp:911]: can you add a todo here documenting why its commented out

Comment (nukeulater) [reply xlive/H2MOD/Modules/Tweaks/Tweaks.cpp:911]: sure

File: xlive/XLive/UserManagement/CUser.cpp (hunk 1 above)
Comment (nukeulater) [reply xlive/XLive/UserManagement/CUser.cpp:430]: self assignment

Comment (pnill) [reply xlive/XLive/UserManagement/CUser.cpp:430]: Is this not assigning the global?

Comment (nukeulater) [reply xlive/XLive/UserManagement/CUser.cpp:430]: No, I checked before removing to make sure it wasn't assigning to something else

**Pull request merged**